File: translations/codenet/mistral_commenetd/Go2J/deepseek/Code/s566920420.java
import java.util.Scanner; // Import the Scanner class for reading input

public class Main {
    public static void main(String[] args) {
        // Create a Scanner object to read input from the standard input stream
        Scanner scanner = new Scanner(System.in);

        // Declare a variable named "a" of type int
        int a;

        // Read a value of type int from the standard input and store it in the variable "a"
        a = scanner.nextInt();

        // Calculate the result of the expression "a + a*a + a*a*a"
        int result = a + a*a + a*a*a;

        // Print the result to the standard output
        System.out.println(result);

        // End of code
    }
}

File: translations/codenet/mistral_commenetd/Go2P/gpt-4o/Code/s566920420.py
# Import the sys module for standard input
import sys

# Read a value of type int from the standard input and store it in the variable "a"
a = int(sys.stdin.readline().strip())

# Calculate the result of the expression "a + a*a + a*a*a"
result = a + a * a + a * a * a

# Print the result to the standard output
print(result)

# End of code
# <END-OF-CODE>
